desktop/apps/collect/client.coffee

```diff
@@ -26,6 +27,7 @@ sd = require('sharify').data
 module.exports.init = ->
   # Set initial params from the url params
   paramsFromUrl = qs.parse(location.search.replace(/^\?/, ''))
+  paramsFromUrl = _.extend paramsFromUrl, aggregations: ['TOTAL', 'FOLLOWED_ARTISTS'], size: 40
```

Comment:
You can probably drop this modification, and update the defaults [here](https://github.com/artsy/force/blob/002a72dfa3a5d4651f69e9f5f7d8731b016b99c7/desktop/components/commercial_filter/models/params.coffee#L26-L34).

Reply:
I'm under the impression the defaults affect the filter component as it is displayed elsewhere (eg on artist, artwork, gene pages, etc). I don't want to change the defaults... just the aggregations on `/collect`. Maybe i'm missing something!

Reply:
oops sorry that was me ^

Reply:
As it turns out, we put most of the views/filters for /collect in that separate components directory, with the _hope_ that it could be reused on other pages.

But, it never made it to any of those pages! So I think the only use of this component (unfortunately we have like 3 separate filter implementations in the codebase...), _is_ actually just on /collect.